packages/node-image-kubernetes/base.packages

```diff
@@ -25,4 +25,4 @@ cray-prodmgr=1.1.2-20220104153307_2374f1f
 cray-sat-podman=2.0.0-1
 
 # SDU
-cray-sdu-rda=2.0.1-shasta_20220712130245_fe07e86
+cray-sdu-rda=2.1.0-shasta_20220810231129_03043fd
```

Comment:
Package does not exist in the defined SDU repo (at this time).

Reply:
updated url